lib/mock-nexmo.js

```diff
@@ -37,8 +52,16 @@ function MockNexmo(log, balanceThreshold) {
 
         log.info({ op: 'sms.send.mock' })
 
-        callback(null, {
-          messages: [{ status: '0' }]
+        // HACK: Enable remote tests to see what was sent
```

Comment:
Is this is still a hack? FWIW, I do like testing the sms this way to receive the code.